e2e/test/scenarios/binning/binning-reproductions.cy.spec.js

```diff
@@ -207,7 +207,9 @@ describe("binning related reproductions", () => {
     cy.findByTestId("sidebar-left").within(() => {
       cy.findByTestId("Table-button").click();
       cy.findByTextEnsureVisible("Table options");
-      cy.findByText("Created At").siblings(".Icon-eye_outline").click();
+      cy.findByText("Created At")
+        .siblings("[data-testid$=hide-button]")
```

Comment:
That's quite a peculiar selector with the $ there.
New to me. Can you explain?

Reply:
It ensures that the `data-testid` ends with that string. The beginning of the testid has the column name in it

Reply:
ah...makes total sense.
Fancy!